apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2023-05-11T19:18:21Z"
-  build_hash: 9e2542cf2c0f92c014524c269474055cca758d70
-  go_version: go1.19.4
-  version: v0.26.0-3-g9e2542c
-api_directory_checksum: 6dad53af842b63621da7311c93b0d29ae0fd2895
+  build_date: "2023-08-24T21:37:01Z"
+  build_hash: e9b68590da73ce9143ba1e4361cebdc1d876c81e
+  go_version: go1.20.3
+  version: v0.26.1-7-ge9b6859
```

Comment:
Maybe in a seperate PR can you regenerate this controller using code-generator `v0.27.0` and pin the ack runtime version to `v0.27.1` in the go.mod? we could also release v0.0.1 in this same PR.

Reply:
sg. releasing 0.0.1 in this PR